static/src/stylesheets/inline/basher.KeepItInTheGround.scss

```diff
@@ -0,0 +1,205 @@
+        padding-left: $gs-gutter;
+        margin-left: -($gs-gutter/2);
+    }
+
+    @include mq(phablet) {
+        height: 300px;
+        padding-top: 0;
+    }
+
+    @include mq(desktop) {
+        display: table-cell;
+        height: 500px;
+        background-size: 130%;
+    }
+}
+
+.keep-it-in-the-ground__copy {
+    position: absolute;
+    top: 0px;
+    left: 10px;
+    padding-right: $gs-gutter/2;
+    margin-right: $gs-gutter;
+
+    @include mq(mobileLandscape) {
+        left: $gs-gutter;
+    }
+
+    @include mq(phablet) {
+        margin-right: 100px;
+    }
+
+    @include mq(leftCol) {
+        left: 170px;
+        border-left: 1px solid $c-text;
+        padding-left: $gs-gutter/2;
+    }
+
+    @include mq(wide) {
+        left: 250px;
+    }
+}
+
+.keep-it-in-the-ground__headline {
+    font-family: $f-serif-headline;
+    font-weight: 200;
+    font-size: 1.5rem;
+    line-height: 1.1;
+    max-width: 700px;
+    color: #ffffff;
+    margin-top: 6px;
+    padding-right: 50px;
+    margin-bottom: 1.5em;
+
+    @include mq(phablet, desktop) {
+        margin-left: 140px;
+        border-left: 1px solid mix(#ffffff, $c-background, 30%);
+        margin-top: 0;
+        padding-top: 6px;
+        padding-left: 10px;
+        padding-right: 150px;
+    }
+
+    @include mq(desktop) {
+        font-size: 2rem;
+        padding-left: $gs-gutter/2;
+    }
+
+    strong {
+        color: $c-accent;
+    }
+}
+
+.keep-it-in-the-ground__button {
+    background-color: $c-accent;
+    border-radius: 50%;
+    width: 22px;
+    height: 22px;
+    display: inline-block;
+    vertical-align: middle;
+
+    @include mq(desktop) {
+        width: 30px;
+        height: 30px;
+    }
+
+    svg {
+        width: 100%;
+        height: auto;
+    }
+}
+
+.keep-it-in-the-ground__credit {
+    font-size: 14px;
+    line-height: 18px;
+    font-family: $f-sans-serif-headline;
+    color: mix(#ffffff, $c-background, 30%);
+
+    @include mq(phablet, desktop) {
+        padding-left: $gs-gutter/2;
+        margin-left: gs-span(2);
+
+        .credit-break {
+            display: none;
+        }
+    }
+
+    @include mq(desktop) {
+        position: absolute;
+        bottom: 0;
+        left: $gs-gutter/2;
+    }
+}
+
+.keep-it-in-the-ground__logo {
+    fill: #ffffff;
+    width: 80px;
+    height: auto;
+    position: absolute;
+    right: 20px;
+    bottom: 0;
+
+    @include mq(phablet, desktop) {
+        left: $gs-gutter;
+        top: 8px;
+        width: 70px;
+    }
+
+    @include mq(desktop) {
+        left: auto;
+        top: auto;
+        width: 100px;
+        right: 20px;
+        bottom: 12px;
+    }
+}
+
+.keep-it-in-the-ground__anchor {
+    position: absolute !important;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 10 !important;
+
+    &:hover ~ .keep-it-in-the-ground__content {
+        strong {
+            color: $c-accent-dark;
+        }
+    
+        .keep-it-in-the-ground__button {
```

Comment:
Not really because of the way layout is.

I could move `.keep-it-in-the-ground__anchor` down a level but it wouldn't cover the entire header.

I can't move the button up a level because it has to follow the text.

On the plus side, it is no js.